.github/workflows/production.yml

```diff
@@ -0,0 +1,40 @@
+name: Push Bot To Prod
+
+# TODO: We need to create dev and prod credentials
+on:
+  push:
+    branches: [develop]
```

Comment:
Why are we pushing from dev? Remember the other comment about clarifying ur intended workflow, maybe should be better add it before to be on the same page about the cli process.

Reply:
There are two differences here, when you push to dev it means that is the only entity that can do it is git instead of the normal user, that's why after the PR when the code is push to the branch we are pushing the resources to azure

Reply:
And when are we updating master then? Because right now sounds like *develop* is the usual master. We are not using something like versions or so to go to production.